DigitalLearningSolutions.Web/Controllers/SignpostingController.cs

```diff
@@ -48,7 +41,7 @@ public async Task<IActionResult> LaunchLearningResource(int resourceReferenceId)
 
             if (delegateUser!.HasDismissedLhLoginWarning)
             {
-                return RedirectToAction("ViewResource", "Signposting", new { resourceReferenceId });
+                return RedirectToAction("Index", "ViewResource", new { resourceReferenceId });

```

Comment:
Additionally there is a `TODO` action that needs checking in related to this endpoint `\\DigitalLearningSolutions.Web\Views\SignpostingSso\LinkLearningHubSso.cshtml`